### PR TITLE
ecr viewer - about the condition (partial)

### LINKED ISSUE #933: eCR Summary - About the Condition (partial)
## Action Requested
Implement part of the "about the condition" section of the Quick eCR Summary - specifically, the "Reportable Condition" field.

## Acceptance Criteria
* "About the Condition" section is included in Quick eCR Summary
* Reportable condition is included
* Input FHIR bundle to eCR insights BB --> receive quick summary that includes "about the condition" 

## Testing
We should test with a few different FHIR bundles, to make sure it works across FHIR bundles generated from different sources.

## Background & Context
This section should be consistent across all eCRs, regardless of condition or source. It originally comes from the RR, so we will need to ensure we're correctly extracting information from the RR during FHIR conversion.

## Additional Information
#930 is a blocker for this work - we should take advantage of a FHIR object library here.

There could be multiple reasons an eCR was triggered, so we may end up with multiple conditions. This will need clarification with design and product, but for this first implementation we should be able to just take the first in the list.

### PR DESCRIPTION
**Pull request by BobanL**
# PULL REQUEST

## Summary
- ~Add new `notes` to the ObservationRRCondition liquid template to include rkms trigger summary that is located in the observation.entryRelationship.organizer.component.observation.entryRelationship.observation.value
image~
- Update the ObservationRRCondition to use [rr-reportability-information-observation profile](http://hl7.org/fhir/us/ecr/StructureDefinition/rr-reportability-information-observation) 
- Add new [extension](http://hl7.org/fhir/us/ecr/StructureDefinition/us-ph-determination-of-reportability-rule-extension) to the observation for the rule that triggered it
- Add all formatting for the `About the Condition` section, but only add data for what is required in the story. The rest of the data is hardcoded with the `(temp)` message attached.

**Updated Screenshot**
image

## Related Issue
Fixes #933 

## Additional Information
Replaced grid-col-auto since long text was moving to a new line underneath rather than wrapping.

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by sarahtress** [comment]
@BobanL this looks good to me! no changes to be made from my end

File: containers/ecr-viewer/src/app/view-data/components/EcrSummary.tsx [hunk 1]
```diff
@@ -24,34 +27,128 @@ const EcrSummary = (
                     <div className="usa-summary-box__text">
                         <div className="grid-row">
                             <div className="data-title"><h4>Patient Name</h4></div>
-                            <div className="grid-col-auto">
```
Comment (JNygaard-Skylight) [inline containers/ecr-viewer/src/app/view-data/components/EcrSummary.tsx:27]: What is the change from `"grid-col-auto"` to `"grid-col-fill"`?

**Review by JNygaard-Skylight — DISMISSED** [review]
Looks good to me, just curious about the change from "grid-col-auto" to "grid-col-fill". It does not seem to do anything to the formatting.

**Comment by sarahtress** [comment]
@BobanL just realized an issue with the spacing sorry! if it's too late just let me know, but basically there just needs to be more spacing above "About the Condition" (should be 20px)

**Review by JNygaard-Skylight — DISMISSED** [review]
Still looks good to me. To fix the spacing @sarahtress mentioned I used this CSS in my branch, but if you have a better solution, feel free to do tha instead:
```
.usa-summary-box__body:not(:first-child) .summary-box-key-information {
  margin-top: 20px;
}
```

**Comment by BobanL** [comment]
> @BobanL just realized an issue with the spacing sorry! if it's too late just let me know, but basically there just needs to be more spacing above "About the Condition" (should be 20px)

Fixed! Take a look at this screenshot and let me know if it's ok
image

**Review by JNygaard-Skylight — APPROVED** [review]

**Comment by sarahtress** [comment]
> > @BobanL just realized an issue with the spacing sorry! if it's too late just let me know, but basically there just needs to be more spacing above "About the Condition" (should be 20px)
> 
> Fixed! Take a look at this screenshot and let me know if it's ok image

@BobanL looks great, thanks!

**Pull request merged**